android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java

```diff
@@ -337,6 +356,43 @@ private void configureAutosaveEnabled(final boolean autosaveEnabled) {
         configuration.autosaveEnabled(autosaveEnabled);
     }
 
+    private void configureAnnotationEditingEnabled(final boolean annotationEditingEnabled) {
+        if (annotationEditingEnabled) {
+            configuration.enableAnnotationEditing();
+        } else {
+            configuration.disableAnnotationEditing();
+        }
+    }
+
+    private void configureEditableAnnotationTypes(@Nullable final ReadableArray editableAnnotationTypes) {
+        if (editableAnnotationTypes == null) {
+            // If explicit null is passed we disable annotation editing.
+            configuration.editableAnnotationTypes(Collections.singletonList(AnnotationType.NONE));
+            return;
+        }
+        List<Object> annotationTypes = editableAnnotationTypes.toArrayList();
+        if (annotationTypes.contains("all")) {
+            // Passing in null enables all annotation types.
+            configuration.editableAnnotationTypes(null);
+            return;
+        }
+
+        // Finally create the actual list of enabled annotation types.
+        List<AnnotationType> parsedTypes = new ArrayList<>();
+        for (Object item : annotationTypes) {
+            String annotationType = item.toString();
+            try {
+                parsedTypes.add(AnnotationType.valueOf(annotationType.toUpperCase(Locale.ENGLISH)));
+            } catch (IllegalArgumentException ex) {
+                Log.w(LOG_TAG,
```

Comment:
Not sure if we should simply skip it in case of an illegal argument. 
If you feel that throwing exception would be too drastic, can we at least log it as an error (`Log.e`)?

Reply:
I wouldn't crash the App but logging an error seems appropriate.